ccd-definition/CaseField/CaseField-COS-nonprod.json

```diff
@@ -48,34 +97,12 @@
     "FieldType": "NotifyClaimCoS",
     "SecurityClassification": "Public"
   },
-  {
-    "CaseTypeID": "CIVIL",
-    "ID": "cosLabelForClaimant1_1",
-    "Label": "Claimant's name <br /> ${applicant1.partyName}",
-    "FieldType": "Label",
-    "SecurityClassification": "Public"
-  },
   {
     "CaseTypeID": "CIVIL",
     "ID": "cosLabelForClaimant1_2",
     "Label": "Claimant's name <br /> ${applicant1.partyName}",
     "FieldType": "Label",
     "SecurityClassification": "Public"
-  },
-  {
```

Comment:
Deleting these varialbe will affect CoS notify screen?

Reply:
these fields are moved up to line 25/32